plugins/ufm_log_analyzer_plugin/src/loganalyze/log_parsing/console_log_regex.py

```diff
@@ -21,33 +21,30 @@
 # as each specific log is being analyzed on his own.
 ###
 
-EXPCT_REGEX = r"expc"
-TRACBEBACK_REGEX = r"traceback"
-
 CONSOLE_LOG_EXCEPTION_REGEX = re.compile(
     # Date and time    NA  log level    The rest of the line
-    r"^([\d\-]+ [\d\:\.]+) \w+ +(\w+) +(.*(?:traceback|exception).*)",
+    r"^([\d\-]+ [\d\:\.]+) \w+ +(\w+) +(.*(?:traceback|exception).*)$",
     re.IGNORECASE,
 )
 
 ERROR_LINE_REGEX = re.compile(
     # Does not start with date   The rest
-    r"^(?![\d\-]+ [\d\:\.]+ \w+)(.*)$",
+    r"^(?!f?)(?![\d\-]+ [\d\:\.]+ \w+)(.*)$"
```

Comment:
Adding the f as it seems some logs that are coming from the telemetry (not written in the UFM code) are doing f<timestamp>